templates/README.md

```diff
@@ -1,41 +1,52 @@
+[coc_policy]: ../policies/code-of-conduct.md
+[COC]: ../CODE_OF_CONDUCT.md
+[COC_TPL]: ../templates/CODE_OF_CONDUCT.md
+[GOV]: ../GOVERNANCE.md
+[GOV_TPL]: ../templates/GOVERNANCE.md
+[CONTRIB]: ../CONTRIBUTING.md
+[mozilla]: https://mozillascience.github.io/working-open-workshop/contributing/
 
-- **CONTRIBUTING.md** to explain how to make successful contributions (be real).
+## Other files
 
-    - If your project uses a DCO, include that in your contribution guide. [For example](https://github.com/hibernate/hibernate-orm/blob/main/CONTRIBUTING.md#legal):
+- If you use a DCO (Developer Certificate of Origin), your project should have a **DCO.txt** (case-insensitive) file containing content from <http://developercertificate.org/>
 
-        ```md
-        All contributions are subject to the [Developer Certificate of Origin (DCO)](http://developercertificate.org/).
-        The DCO text is also included verbatim in the [dco.txt](./dco.txt) file in the root directory of this repository.
-        ```
+## Contributor Agreements
 
-    - If your project uses a CLA, include that requirement in the contribution guide. [For example](https://github.com/jreleaser/jreleaser/blob/main/CONTRIBUTING.adoc#contributor-license-agreement):
+Explanation of the difference between DCO and CLA: <https://www.consortiuminfo.org/open-source/all-about-clas-and-dcos/>
 
-        ```md
-        Contributor License Agreement
+- If your project uses a DCO, include that in your contribution guide. [For example](https://github.com/hibernate/hibernate-orm/blob/main/CONTRIBUTING.md#legal):
 
-        Contributions to JReleaser are protected by a CLA.
-        Please read the document before making a contribution.
-        You’ll be asked to digitally sign the document on your first contribution.
-        Feel free to open a discussion topic if you have questions.
-        ```
+    ```md
+    All contributions are subject to the [Developer Certificate of Origin (DCO)](http://developercertificate.org/).
+    The DCO text is also included verbatim in the [dco.txt](./dco.txt) file in the root directory of this repository.
+    ```
 
-    - Explanation of the difference between DCO and CLA: <https://www.consortiuminfo.org/open-source/all-about-clas-and-dcos/>
+- If your project uses a CLA, include that requirement in the contribution guide. [For example](https://github.com/jreleaser/jreleaser/blob/main/CONTRIBUTING.adoc#contributor-license-agreement):
 
-- **CODEOWNERS** to define individuals or teams that are responsible for code in a repository
-    - [About Code Owners (GitHub)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
-    - [Code Owners (GitLab)](https://docs.gitlab.com/ee/user/project/codeowners/)
+    ```md
+    Contributor License Agreement
+
+    Contributions to JReleaser are protected by a CLA.
```

Comment:
Is it intended to mention JRelease here as an example or it should be MY_PROJECT or something like this?

Reply:
This is an example. Happy for you to suggest the change to make it more readable / more obviously an example.

Reply:
`Contributions to My_Cool_Commonhaus_Project`?

Reply:
Sure. I will train you yet.. use the +- thing so I can just commit your suggestion. ;)

Reply:
```suggestion
    Contributions to JReleaser are protected by a CLA.
```